fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -365,9 +365,7 @@ private void refreshCardinality() {
             Set<ApiFilter> filters,
             PaginationParameters paginationParameters
     ) {
-        int perPage = paginationParameters.getPerPage();
-        validatePerPage(perPage);
```

Comment:
Also, the fact that removing this didn't break any tests means that we're missing some tests around this...